modules/apps/portal-vulcan/portal-vulcan-impl/src/main/java/com/liferay/portal/vulcan/internal/template/servlet/RESTClientHttpRequest.java

```diff
@@ -51,6 +59,11 @@ public Enumeration<String> getHeaders(String name) {
 				Arrays.asList(ContentTypes.APPLICATION_JSON));
 		}
 
+		if (StringUtil.equalsIgnoreCase(name, "Accept-Language")) {
+			return Collections.enumeration(
+				Arrays.asList(_locale.toLanguageTag()));
```

Comment:
I think Collections.enumeration is not necessary as a List already implements a Collection interface